12-remote-sensing-systems.Rmd

```diff
@@ -406,9 +406,9 @@ Due to the unusual shape of the lens, it captures a much larger proportion of a
 
 ```{r 12-hemispherical-lens, fig.cap = fig_cap, out.width="90%", echo = FALSE}
 
-fig_cap <- paste("[Hemispherical photo taken in the Bavarian forest](https://en.wikipedia.org/wiki/File:Hemispherical_photo1.jpg). [Martin Wegmann](https://commons.wikimedia.org/wiki/User:Wegmann). Animated figure can be viewed in the web browser version of the textbook. [CC BY 3.0 Unported](https://creativecommons.org/licenses/by/3.0/)")
+fig_cap <- paste("[Hemispherical photo taken in the Bavarian forest](https://en.wikipedia.org/wiki/File:Hemispherical_photo1.png). [Martin Wegmann](https://commons.wikimedia.org/wiki/User:Wegmann). Animated figure can be viewed in the web browser version of the textbook. [CC BY 3.0 Unported](https://creativecommons.org/licenses/by/3.0/)")
```

Comment:
@natasha-sharma123 This will have to be reverted because it's not a path that was changed, and the link to wikipedia will be broken